doc/api_assets/style.css

```diff
@@ -339,6 +339,8 @@ p code,
 li code {
   font-size: 0.9em;
   color: #040404;
+  background-color: #f2f5f0;
+  padding: 0.2em;
```

Comment:
I think a padding of like `0.2em 0.4em` would look better visually, it's what GitHub does too (They do it through a unicode space in `:before` and `:after`).